api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -198,6 +198,7 @@ private Patient createBasicPatient() {
 		// patient.removeAddress(pAddress);
 		
 		patient.setDeathDate(new Date());
+		patient.setDead(true);
 		// patient.setCauseOfDeath("air");
 		patient.setBirthdate(new Date());
 		patient.setBirthdateEstimated(true);
```

Comment:
I would also expect you to add some new tests for the functionality you are putting in place.